tests/unit/utils_license.cpp

```diff
@@ -139,3 +137,20 @@ TEST_F(LicenseTest, LicenseInfoOverride) {
     CheckLicenseValidity(false);
   }
 }
+
+TEST_F(LicenseTest, LicenseType) {
+  CheckLicenseValidity(false);
+  const std::string organization_name{"Memgraph"};
+  {
+    memgraph::license::License license_entr{organization_name, 0, 0, memgraph::license::LicenseType::ENTERPRISE};
+    const std::string license_key = memgraph::license::Encode(license_entr);
+    license_checker->SetLicenseInfoOverride(license_key, organization_name);
+    CheckLicenseValidity(true);
+  }
+  {
+    memgraph::license::License license_oem{organization_name, 0, 0, memgraph::license::LicenseType::OEM};
+    const std::string license_key = memgraph::license::Encode(license_oem);
+    license_checker->SetLicenseInfoOverride(license_key, organization_name);
```

Comment:
Maybe use "random numbers" instead of `0`, because it might interfere with the license type in a strange way (faulty serializer, deserializer) and it might not be caught.